cloud/linode/client/client.go

```diff
@@ -39,7 +39,7 @@ type Client interface {
 // linodego.Client implements Client
 var _ Client = (*linodego.Client)(nil)
 
-func newLinodeClient(token, ua, apiURL string) (*linodego.Client, error) {
+func NewLinodeClient(token, ua, apiURL string) (*linodego.Client, error) {
```

Comment:
Since this function is now exported, I would recommend adding some godoc.

This is a bit of a nit, but `client.NewLinodeClient` kind of stutters. Maybe calling the function `New` would flow a bit more nicely.

Lastly, please expand the `ua` argument to `userAgent`, so that even without godoc, the argument's purpose is clear.